.github/ISSUE_TEMPLATES/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
Also add:
`<!-- Remove anything that is not applying to your issue/bug context -->`

Reply:
@Abhishek-kumar09 done